CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 ## dev
 
-Version <dev> fixes a bug related to expected errors not bearing a "true" value for the "expected" attribute if expected as a result of an HTTP status code match and changes the way Stripe instrumentation metrics are named to prevent high-cardinality issues.
+Version <dev> introduces instrumentation for the aws-sdk-sqs gem, fixes a bug related to expected errors not bearing a "true" value for the "expected" attribute if expected as a result of an HTTP status code match and changes the way Stripe instrumentation metrics are named to prevent high-cardinality issues.
```

Comment:
Sometimes we link gems and I always kinda liked that
```suggestion
Version <dev> introduces instrumentation for the [aws-sdk-sqs gem](https://rubygems.org/gems/aws-sdk-sqs/versions/1.0.0.rc11), fixes a bug related to expected errors not bearing a "true" value for the "expected" attribute if expected as a result of an HTTP status code match and changes the way Stripe instrumentation metrics are named to prevent high-cardinality issues.
```